platform/net/ubuntu_net_manager.go

```diff
@@ -359,15 +359,17 @@ auto lo
 iface lo inet loopback
 {{ range .DHCPConfigs }}
 auto {{ .Name }}
-iface {{ .Name }} inet dhcp
+iface {{ .Name }} inet dhcp{{ range .PostUpRoutes }}
+post-up route add -net {{ .Destination }} netmask {{ .NetMask }} gw {{ .Gateway }}{{ end }}
 {{ end }}{{ range .StaticConfigs }}
 auto {{ .Name }}
 iface {{ .Name }} inet{{ .Version6 }} static
     address {{ .Address }}{{ if not .IsVersion6 }}
     network {{ .Network }}{{ end }}
     netmask {{ .NetmaskOrLen }}{{ if .IsDefaultForGateway }}{{ if not .IsVersion6 }}
     broadcast {{ .Broadcast }}{{ end }}
-    gateway {{ .Gateway }}{{ end }}
+    gateway {{ .Gateway }}{{ end }}{{ range .PostUpRoutes }}
+    post-up route add -net {{ .Destination }} netmask {{ .NetMask }} gw {{ .Gateway }}{{ end }}
```

Comment:
does this work for IPv6?

Reply:
I thought it does work for IPv4 now. Because the IPv6 static route entry is different from IPv4, the  typical IPv^ configuration is that `up ip -6 route add ::/0 via xxxx:xxxx::yyyy dev eth0`.

Reply:
Can we address both. BOSH team is working on adding IPv6 support right now...